README.md

```diff
@@ -14,24 +14,21 @@ add the `UnverifiedMox` module to your checks and configure it to only include t
 # ... .credo.exs
   checks: [
     ## other Credo checks...
-    {CredoMox.Checks.UnverifiedMocks, files: %{included: ["**/*_test.exs"]}},
+    {CredoMox.Checks.UnverifiedMox, files: %{included: ["**/*_test.exs"]}},
   ]
 ```
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `credo_mox` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `credo_mox` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:credo_mox, "~> 0.1.0"}
+    {:credo_mox, "~> 0.1", only: [:dev, :test], runtime: false},
```

Comment:
That was really what I was going for originally -- didn't want users to have credo mox loaded in their production apps.  Thank you for the changes here 👍